platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulers.java

```diff
@@ -16,19 +16,46 @@
 
 package com.swirlds.platform.wiring;
 
+import static com.swirlds.common.wiring.model.HyperlinkBuilder.platformCommonHyperlink;
+import static com.swirlds.common.wiring.model.HyperlinkBuilder.platformCoreHyperlink;
+
 import com.swirlds.common.context.PlatformContext;
+import com.swirlds.common.stream.EventStreamManager;
```

Comment:
This fails with

```
/runner/_work/hedera-services/hedera-services/platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulers.java:23: error: cannot find symbol
import com.swirlds.common.stream.EventStreamManager;
                                ^
  symbol:   class EventStreamManager
  location: package com.swirlds.common.stream
```

https://github.com/hashgraph/hedera-services/actions/runs/8381299589/job/22952492704#step:6:4673

Reply:
I take that back. This seems to be a regression of https://github.com/hashgraph/hedera-services/commit/6e1dc367c629e212932e4163d691b1f64e69329c.